Gemfile

```diff
@@ -77,3 +77,6 @@ end
 group :development, :test do
   gem 'rspec-rails'
 end
+
+gem 'bullet', group: 'development'
+gem 'xmpp4r', group: 'development'
```

Comment:
- [ ] As required for the project, kindly ensure that you don't use the `bullet gem` in this exercise. Instead analyze the logs as showed in this [article](https://dev.to/junko911/rails-n-1-queries-and-eager-loading-10eh). After that, you can update your images `before` and `after` solving  the N+1 problem in your PR description without using the `bullet gem`